src/act/node/canister_method/update_method.rs

```diff
@@ -41,13 +44,13 @@ impl Deref for UpdateMethod {
 
 impl Declare<Context> for UpdateMethod {
     fn to_declaration(&self, context: &Context, _: String) -> Option<Declaration> {
+        let user_defined_name = &self.name;
         let function_declaration = self.generate_function_body(context);
-
         let macro_args = self.generate_macro_args(&context.cdk_name);
 
         Some(quote! {
             #[ic_cdk_macros::update(#macro_args)]
```

Comment:
Don't we have to rename this here? Otherwise the Wasm import will be incorrect right?

Reply:
Yes. It's renamed inside of `#macro_args`